daemon/logger/loggerutils/rotatefilewriter.go

```diff
@@ -1,23 +1,34 @@
 package loggerutils
 
 import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
 	"errors"
+	"io"
 	"os"
 	"strconv"
 	"sync"
 
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/jsonlog"
 	"github.com/docker/docker/pkg/pubsub"
 )
 
+const defaultTimestamp = "0001-01-01T00:00:00Z"
```

Comment:
If you don't prefer an empty string here, 1970-01-01T00:00:00Z would be consistent with most other programs.

PTAL @cpuguy83 WDYT?